src/vue/pages/FundCreationRequests.vue

```diff
@@ -232,13 +235,24 @@ export default {
     },
   },
 
+  created () {
+    this.initHistoryLoading()
+  },
+
   methods: {
-    async getHistory () {
-      const response = await Sdk.horizon.request.getAllForSales({
-        requestor: this.account.accountId,
-      })
+    initHistoryLoading () {
```

Comment:
confusing name

Reply:
What about splitting this method into two methods: `setRequestLoader` and `setInitialLoadingState` (`resetData`, `setLoadingState` or smth else)? Any ideas?

Reply:
Renamed to `initFirstPageLoader`